main/graphite2/APKBUILD

```diff
@@ -2,7 +2,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=graphite2
 _realname=${pkgname/2/}
-pkgver=1.3.12
+pkgver=1.3.13
 pkgrel=1
 pkgdesc="reimplementation of the SIL Graphite text processing engine"
 url="http://graphite.sil.org/"
```

Comment:
new url is https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_home

Reply:
graphite.sil.org redirects there.
I'd rather keep it as-is (or, rather, changes to a minimum) since this isn't my package.